src/function_run_result.rs

```diff
@@ -20,6 +20,7 @@ pub struct FunctionRunResult {
     pub runtime: Duration,
     pub size: u64,
     pub memory_usage: u64,
+    pub fuel_consumed: u64,
```

Comment:
If this is the name that will output with the `--json` arg, can we update it to instructions here too?